examples/viewer.py

```diff
@@ -73,6 +76,50 @@ def __init__(self, sim_settings: Dict[str, Any]) -> None:
             key.Z: "move_up",
         }
 
+        # Load a TrueTypeFont plugin and open the font
+        self._font = text.FontManager().load_and_instantiate("TrueTypeFont")
+        self._font.open_file(
+            os.path.join(os.path.dirname(__file__), "Cousine-Regular.ttf"), 180.0
+        )
+
+        # Glyphs we need to render everything
+        self._cache = text.DistanceFieldGlyphCache(
+            mn.Vector2i(2048), mn.Vector2i(512), 22
+        )
+        self._font.fill_glyph_cache(
+            self._cache,
+            "abcdefghijklmnopqrstuvwxyz"
+            "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+            "0123456789:-_+,.! %µ",
```

Comment:
Wouldn't this be sufficient for the glyph cache?
```suggestion
            string.printable,
```

Reply:
Yeah probably, I can check it out. I was just using the code from the magnum example page, so there are probably improvements that can be made for our specific purposes

Reply:
There's (currently) an issue where if you put too many glyphs in, it silently fails to populate the glyph cache. I'll fix that eventually but for now I'd say safest is to have the characters strictly enumerated like this. Once I have that fixed, I can clean this code up.

Reply:
Can still be simplified using
```suggestion
string.ascii_lowercase + string.ascii_uppercase +  string.digits + ":-_+,.! %µ"
```

Reply:
Fo sho. I will make that change